requirements.txt

```diff
@@ -32,3 +32,5 @@ stups-tokens>=1.0.14
 subprocess32
 timeperiod2==0.1
 Yapsy>=1.10.423
+opentracing==1.2.2
+basictracer>=2.2.0
```

Comment:
why have it in when the --opentracing help says `Enable opentracing with one of supported providers: "instana", "basic"`?

Reply:
True. I think basictracer is not a dependency here.

Reply:
BasicTracer with default initialisation is basically noop: https://github.com/opentracing/basictracer-python/blob/master/basictracer/tracer.py#L27

Reply:
Still not a dependency for zmon-worker 😉